istioctl/pkg/verifier/verifier.go

```diff
@@ -94,6 +96,13 @@ func (v *StatusVerifier) Verify() error {
 }
 
 func (v *StatusVerifier) verifyInstallIOPRevision() error {
+	var err error
+	if v.controlPlaneOpts.Revision == "" {
+		v.controlPlaneOpts.Revision, err = getRevision(v.istioNamespace, v.kubeconfig, v.context)
```

Comment:
Could it be possible istio is installed in other namespace rather than `istio-system`?